foreman-selinux/foreman-selinux.spec

```diff
@@ -130,6 +131,7 @@ Summary: SELinux policy module for Foreman Proxy
 Group:   System Environment/Base
 
 Requires:           selinux-policy >= %{selinux_policy_ver}
+Requires:           container-selinux >= 2.0
```

Comment:
No reason I can think of for requiring the package, please explain why you think you need it?